spark/src/main/scala/io/delta/tables/execution/VacuumTableCommand.scala

```diff
@@ -68,4 +56,7 @@ case class VacuumTableCommand(
     }
     VacuumCommand.gc(sparkSession, deltaLog, dryRun, horizonHours).collect()
   }
+
+  override protected def withNewChildInternal(newChild: LogicalPlan): LogicalPlan =
```

Comment:
nit: maybe put this method at the top with the other `LogicalPlan` overrides? Easy to miss down here at the bottom...

Reply:
Done